### DISCUSSION TITLE
Fixed sort menu

### DISCUSSION THREAD

**Discussion by aleqske** [dailydotdev/daily#451] [Feature Requests]
It's great that now we can sort articles by Date and Recommended. However, it's kinda difficult to sort or switch your sort preference when you've already scrolled down the page because the sort menu gets hid as one scrolls down. I'm aware that there is a 'Back to top' link but I feel making this sort menu always visible is somehow user friendly.

It would be great if this menu stayed fixed at the top to allow for ease selection no matter where one is on the page.

**Comment by rebelchris** [comment]
Thanks for creating this discussion!
I'll ask the team what they think ✨
